extensions/ruby/extension.toml

```diff
@@ -8,15 +8,15 @@ repository = "https://github.com/zed-industries/zed"
 
 [language_servers.solargraph]
 name = "Solargraph"
-language = "Ruby"
+languages = ["Ruby"]
 
 [language_servers.ruby-lsp]
 name = "Ruby LSP"
-language = "Ruby"
+languages = ["Ruby", "ERB"]
 
 [language_servers.rubocop]
 name = "Rubocop"
-language = "Ruby"
+languages = ["Ruby"]
```

Comment:
Rubocop does not support because there is [`erb-lint`](https://github.com/Shopify/erb_lint) linter that utilises `rubocop` under the hood.